src/font-faces.css

```diff
@@ -9,68 +9,274 @@
  * reduces wait time for fonts that we know we are using.
  */
 
-/*
+/* ––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
+Font: Circular
+Cuts: Thin, Thin Italic, Light, Light Italic, Regular, Italic, Book, Book Italic, Medium, Medium Italic, Bold, Bold Italic, Black, Black Italic, ExtraBlack, ExtraBlack Italic
+Format: woff, woff2
+
+Purchase Date: November 5, 2019
+––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
+
+Legal Disclaimer for Web licence
+
+Licence to use self-hosted webfonts for displaying dynamic text on specified website domains. Our package includes WOFF and WOFF2 font formats.
+
+Should the license page views limit be exceeded the license holder will be required to contact Lineto.com for an upgrade.
+
+It is strictly prohibited to rename the font and to download or use these fonts in any other media.
+
+These Web fonts are licensed exclusively for the use on the following domain(s) and their subdomains:
+
+sharetribe.com (Monthly Pageviews: <250K)
+
+––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
+
+INSTRUCTIONS
+
+The Web Font delivery folder contains all purchased Web Fonts, example CSS
+file containing @font-face feature statements, and HTML Sample Pages.
+You are only allowed to use the cuts listed above for the licensed
+application.
+
+We recommend using the @font-face statements from the example CSS file.
+Depending on your application structure, you may need to adjust the
+paths to the fonts. It is prohibited to change font/file names.
+
+Please see the HTML Sample Page in the font delivery package for further
+information about Character Groups, Language Support and
+OpenType features etc.
+
+––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
+
+Manufacturer  Lineto GmbH
+
+Vendor URL    https://lineto.com
+
+Copyright     Copyright © 2019 by Lineto GmbH.
+              https://lineto.com
+              All rights reserved
+
+Designer
+              https://lineto.com/designers
+
+EULA          https://lineto.com/assets/pdf/EULA.pdf
+
+*/
+
+
 @font-face {
-  font-family: CircularStd;
-  src: url("./font-files/circular-std/CircularStd-Black.woff2") format("woff2");
-  font-weight: 800;
+  font-family: "Circular";
+  src: url("https://assets-sharetribecom.sharetribe.com/webfonts/circular/CircularXXSub-ThinSubset.woff2") format("woff2");
+  font-weight: 100;
   font-style: normal;
-  font-display: fallback;
```

Comment:
Is there a specific reason to remove the font display behavior? I remember spending time fine-tuning this based on some best practices. I understand that first-time flashing of fallbacks might be annoying, but there are accessibility reasons at play here.

Reply:
Thanks for this @kpuputti - I totally missed this by accident as I only copied the CSS from the font-provider and it didn't have the font-display: fallback in it. Totally new thing for me, I'll add it back. Good catch 🍬